descriptors/postgres/debezium-postgres-1.9.0.Alpha1.json

```diff
@@ -72,83 +72,6 @@
-          "x-category" : "CONNECTION_ADVANCED_SSL"
-        },
-        "database.sslpassword" : {
-          "title" : "SSL Client Key Password",
-          "description" : "Password to access the client private key from the file specified by 'database.sslkey'. See the Postgres SSL docs for further information",
-          "oneOf" : [ {
-            "format" : "password",
-            "description" : "Password of the database user to be used when connecting to the database.",
-            "type" : "string"
-          }, {
-            "description" : "An opaque reference to the password.",
-            "type" : "object",
-            "properties" : { },
-            "additionalProperties" : true
-          } ],
-          "x-name" : "database.sslpassword",
-          "x-category" : "CONNECTION_ADVANCED_SSL"
-        },
-        "database.sslrootcert" : {
-          "title" : "SSL Root Certificate",
-          "description" : "File containing the root certificate(s) against which the server is validated. See the Postgres JDBC SSL docs for further information",
-          "type" : "string",
-          "x-name" : "database.sslrootcert",
-          "x-category" : "CONNECTION_ADVANCED_SSL"
-        },
-        "database.sslkey" : {
-          "title" : "SSL Client Key",
-          "description" : "File containing the SSL private key for the client. See the Postgres SSL docs for further information",
-          "type" : "string",
-          "x-name" : "database.sslkey",
-          "x-category" : "CONNECTION_ADVANCED_SSL"
-        },
-        "database.sslfactory" : {
-          "title" : "SSL Root Certificate",
-          "description" : "A name of class to that creates SSL Sockets. Use org.postgresql.ssl.NonValidatingFactory to disable SSL validation in development environments",
-          "type" : "string",
-          "x-name" : "database.sslfactory",
-          "x-category" : "CONNECTION_ADVANCED_SSL"
-        },
-        "database.tcpKeepAlive" : {
-          "title" : "TCP keep-alive probe",
-          "description" : "Enable or disable TCP keep-alive probe to avoid dropping TCP connection",
-          "default" : true,
-          "type" : "boolean",
-          "x-name" : "database.tcpKeepAlive",
-          "x-category" : "CONNECTION_ADVANCED"
-        },
-        "database.initial.statements" : {
-          "title" : "Initial statements",
-          "description" : "A semicolon separated list of SQL statements to be executed when a JDBC connection to the database is established. Note that the connector may establish JDBC connections at its own discretion, so this should typically be used for configurationof session parameters only, but not for executing DML statements. Use doubled semicolon (';;') to use a semicolon as a character and not as a delimiter.",
-          "type" : "string",
-          "x-name" : "database.initial.statements",
-          "x-category" : "CONNECTION_ADVANCED"
-        },
-        "plugin.name" : {
```

Comment:
@gunnarmorling I think `plugin.name` should be included?

Reply:
@gunnarmorling what is with that one?

Reply:
No, let's just go with pgoutput as the only supported one. wal2json is unsupported soon anyways, and decoderbufs isn't needed here either.